src/Feliz.MaterialUI/ThemeOverrides.fs

```diff
@@ -1925,7 +1925,7 @@ module themeOverrides =
         static member inline centered(styles: IStyleAttribute list) : IThemeProp = unbox ("overrides.MuiTabs.centered", createObj !!styles)
         /// Styles applied to the tablist element.
         static member inline scroller(styles: IStyleAttribute list) : IThemeProp = unbox ("overrides.MuiTabs.scroller", createObj !!styles)
-        /// Styles applied to the tablist element if `!variant="scrollable"`.
+        /// Styles applied to the tablist element if `!variant="scrollable"`.
```

Comment:
The IDE was complaining there was an invisible character here invalidating the XML doc.

Reply:
I see. This will be reverted the next time the generator runs, though. I'll see about fixing it there.